dsls/sql/src/main/java/org/apache/beam/dsls/sql/rel/BeamJoinRel.java

```diff
@@ -242,17 +266,22 @@ private BeamSqlRow buildNullRow(BeamRelNode relNode) {
     return nullRow;
   }
 
-  private List<Pair<Integer, Integer>> extractJoinColumns(int separator) {
+  private List<Pair<Integer, Integer>> extractJoinColumns(int leftRowColumnCount) {
+    // it's a CROSS JOIN because: condition == true
+    if (condition instanceof RexLiteral && (Boolean) ((RexLiteral) condition).getValue()) {
```

Comment:
For the following cross join:

```sql
select * from person p1, person p2
```

its relation algebra tree is:

```
LogicalProject(id=[$0], name=[$1], age=[$2], ts=[$3], boolean_c=[$4], id0=[$5], name0=[$6], age0=[$7], ts0=[$8], boolean_c0=[$9])
  LogicalJoin(condition=[true], joinType=[inner])
    EnumerableTableScan(table=[[person]])
    EnumerableTableScan(table=[[person]])
```

so in Calcite, a `CROSS JOIN` is converted to a `INNER JOIN` with the `condition` set to `true`.